### ISSUE TITLE
Respond to iOS 15 per-app text size

### ISSUE THREAD

**Issue by MaybeThisIsRu** [snikket-im/snikket-ios#184]
**Is your feature request related to a problem? Please describe.**
It's super difficult for me to use Snikket because I just don't like small text on small screens in general. Everything is bumped to 120% on my iPhone, but Snikket fails to respond to per-app settings, and I believe global settings as well. This is super important for accessibility.

In the future, I hope more padding can be added as well as typography can be improved, but for now, simply being able to bump up the text size will be a good starting point.

**Additional context**
- [How to change your iPhone’s text size for a specific app: In iOS 15, you can have a different text size for each app](https://www.theverge.com/22580423/ios-15-iphone-text-size-how-to-control-center)
- From https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/
  > Be prepared for text-size changes. People expect most apps to respond when they choose a different text size in Settings. To accommodate some text-size changes, you might need to adjust the layout. For more information about text usage in your app, see [Typography](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/).

---

Many established apps don't respond to this either. Element and Telegram for example don't. Would be lovely for XMPP and Snikket to get ahead of it. ;)

**Comment by mwild1** [comment]
I confirmed that this is working with the latest version of the app :+1: 

Thanks for the report!
